src/js/components/Card/__tests__/Card-test.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { cleanup } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import 'jest-styled-components';
+
+import { Avatar } from '../../Avatar';
+import { Grommet } from '../../Grommet';
+import { Card } from '../Card';
+
+const src = '';
+
+describe('Card', () => {
+  afterEach(cleanup);
+
+  test('renders', () => {
+    const component = renderer.create(
+      <Grommet>
+        <Card id="test id" name="test name" />
```

Comment:
I don't think Card should need `id` or `name`. Can we prune this down to just `<Card />`?